packages/orchestrator/nodemon.json

```
@@ -0,0 +1,6 @@
+{
+    "watch": ["lib", "../utils/dist", "../types/dist", "../scheduler/dist", "../../.env"],
```

note sure there is a good reason we listen to lib, it restart too soon all the time but maybe I'm missing something
```suggestion
    "watch": ["dist", "../utils/dist", "../types/dist", "../scheduler/dist", "../../.env"],
```